sanskrit_parser/util/DhatuWrapper.py

```diff
@@ -40,95 +45,97 @@ def _get_file(self):
         if not os.path.exists(os.path.join(self.base_dir, filename)):
             self.logger.debug("%s not found. Downloading it", filename)
             r = requests.get(self.git_url, stream=True)
-            assert r.status_code != 404,\
-                   "Could not download file {}".format(self.git_url)
+            assert r.status_code != 404, \
+                "Could not download file {}".format(self.git_url)
             with open(os.path.join(self.base_dir, filename), "wb") as fd:
                 for chunk in r.iter_content(chunk_size=128):
                     fd.write(chunk)
-        
+
     def _generate_db(self):
         """ Create db from tsv file """
         self.logger.debug("Parsing files into dict for faster lookup")
-        with open(os.path.join(self.base_dir, self.local_filename),"r") as csvfile:
+        with open(os.path.join(self.base_dir, self.local_filename), "r") as csvfile:
             reader = csv.reader(csvfile, delimiter='\t', quotechar='"')
             # FIXME - Rewrite from here
-            for irx,row in enumerate(reader):
+            for irx, row in enumerate(reader):
                 # Get Keys
-                if irx==0:
+                if irx == 0:
                     # Only the first 9 columns are useful
-                    headers=[SanskritObject(x).transcoded(SLP1) for x in row[0:8]] 
+                    headers = [SanskritObject(x).transcoded(SLP1) for x in row[0:8]]
                     self.logger.debug("Found dhatu tsv headers: {}".format(str(headers)))
                 else:
                     j = {}
-                    for i,x in enumerate(row):
-                        if i <len(headers):
-                            t=SanskritObject(x).transcoded(SLP1)
-                            j[headers[i]]=t
+                    for i, x in enumerate(row):
+                        if i < len(headers):
+                            t = SanskritObject(x).transcoded(SLP1)
+                            j[headers[i]] = t
                     self.db.insert(j)
         self.logger.debug("Saved dhatus database")
 
- 
-    def _get_dhatus(self,d):
-        " Get all tags for a dhatu d "
+    def _get_dhatus(self, d):
+        """ Get all tags for a dhatu d """
         if d is None:
             return None
         else:
-            return self.db.search(self.q.DAtuH==d)
-        
-    def is_sakarmaka(self,d):
-        " Is d sakarmaka? "
+            return self.db.search(self.q.DAtuH == d)
+
+    def is_sakarmaka(self, d):
+        """ Is d sakarmaka? """
         # Tags
-        tl=self._get_dhatus(d)
-        if len(tl)!=0:
-            r=reduce(lambda x,y: x or y,['sakarmaka' in t['karmakatvaM'] for t in tl])
+        tl = self._get_dhatus(d)
+        if len(tl) != 0:
+            r = functools.reduce(lambda x, y: x or y, ['sakarmaka' in t['karmakatvaM'] for t in tl])
```

Comment:
I think a simpler option would be to use the builtin any(), i.e. any(['sakarmaka' in t['karmakatvaM'] for t in tl])

Reply:
Yes, that's a lot simpler. I didn't think of that in the first place!

Reply:
While we're at it, might as well fix #69 too?

Reply:
Makes sense. Fixed with 7534955.